Description: Disable newly-introduced *B1NRM2 tests
 They fail on i386.
 See https://buildd.debian.org/status/fetch.php?pkg=lapack&arch=i386&ver=3.12.0-1&stamp=1702247469&raw=0
Author: Sébastien Villemot <sebastien@debian.org>
Bug: https://github.com/Reference-LAPACK/lapack/issues/963
Forwarded: not-needed
Last-Update: 2024-01-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: lapack/BLAS/TESTING/cblat1.f
===================================================================
--- lapack.orig/BLAS/TESTING/cblat1.f
+++ lapack/BLAS/TESTING/cblat1.f
@@ -257,9 +257,6 @@
    20       CONTINUE
             IF (ICASE.EQ.6) THEN
 *              .. SCNRM2 ..
-*              Test scaling when some entries are tiny or huge
-               CALL CB1NRM2(N,(INCX-2)*2,THRESH)
-               CALL CB1NRM2(N,INCX,THRESH)
 *              Test with hardcoded mid range entries
                CALL STEST1(SCNRM2(N,CX,INCX),STRUE2(NP1),STRUE2(NP1),
      +                     SFAC)
Index: lapack/BLAS/TESTING/dblat1.f
===================================================================
--- lapack.orig/BLAS/TESTING/dblat1.f
+++ lapack/BLAS/TESTING/dblat1.f
@@ -320,9 +320,6 @@
 *
             IF (ICASE.EQ.7) THEN
 *              .. DNRM2 ..
-*              Test scaling when some entries are tiny or huge
-               CALL DB1NRM2(N,(INCX-2)*2,THRESH)
-               CALL DB1NRM2(N,INCX,THRESH)
 *              Test with hardcoded mid range entries
                STEMP(1) = DTRUE1(NP1)
                CALL STEST1(DNRM2(N,SX,INCX),STEMP(1),STEMP,SFAC)
Index: lapack/BLAS/TESTING/sblat1.f
===================================================================
--- lapack.orig/BLAS/TESTING/sblat1.f
+++ lapack/BLAS/TESTING/sblat1.f
@@ -320,9 +320,6 @@
 *
             IF (ICASE.EQ.7) THEN
 *              .. SNRM2 ..
-*              Test scaling when some entries are tiny or huge
-               CALL SB1NRM2(N,(INCX-2)*2,THRESH)
-               CALL SB1NRM2(N,INCX,THRESH)
 *              Test with hardcoded mid range entries
                STEMP(1) = DTRUE1(NP1)
                CALL STEST1(SNRM2(N,SX,INCX),STEMP(1),STEMP,SFAC)
Index: lapack/BLAS/TESTING/zblat1.f
===================================================================
--- lapack.orig/BLAS/TESTING/zblat1.f
+++ lapack/BLAS/TESTING/zblat1.f
@@ -257,9 +257,6 @@
    20       CONTINUE
             IF (ICASE.EQ.6) THEN
 *              .. DZNRM2 ..
-*              Test scaling when some entries are tiny or huge
-               CALL ZB1NRM2(N,(INCX-2)*2,THRESH)
-               CALL ZB1NRM2(N,INCX,THRESH)
 *              Test with hardcoded mid range entries
                CALL STEST1(DZNRM2(N,CX,INCX),STRUE2(NP1),STRUE2(NP1),
      +                     SFAC)
